### DISCUSSION TITLE
[ FF7 ] Batttle Animations are too fast for certain characters

### DISCUSSION THREAD

**Discussion by corysnave** [julianxhokaxhiu/FFNx#532] [Q&A] [answered]
**PLEASE NOTE:** If you come here to report a crash while you were using 7th Heaven, please do on their relative support channels. NO SUPPORT will be provided, and your issue will be close immediately.

**Describe the bug**
Very recently I have had an issue with battle animations while running the 60/30 FPS Gameplay mod. I am not sure if it is was caused it but I recently updated to 7th Heaven 3.0/FFNx 1.15. When I turn the mod off the animations go back to normal but obviously my FPS is fucked. I have also tried rolling back to FFNx 1.14 but to no avail. The odd thing is is that it seems to be character specific. See linked video where Barrett's animations seem normal but Cloud and Tifa's are too fast. I have also tried the speedhack settings. Turning them on/off and increasing/decreasing the setting and it has no effect. I have tried a clean install and clean mod install to no avail. Any assistance is greatly appreciated.

**To Reproduce**
Go into any battle 

**Expected behavior**
Animations are the expected speed

**Screenshots**
Video of animations speed for Tifa, Barret, and Cloud
https://imgur.com/a/JfpB8T6

Installed mods
https://imgur.com/a/Z7slBgF

**GPU (please complete the following information):**

Nvidia 3080
Drivers: 528.24 (also recently updated)

**Additional context**

**Comment by julianxhokaxhiu** [comment, marked as answer]
Thanks for having reported the issue, but this is not a core FFNx issue, and as stated above anything related to 7th has their own support channel. Please consider using those to report issues.

Regarding your specific one, the order of mods is totally wrong and you need to use the `1/9` button on the right side to autosort them.

Good luck and enjoy FFNx!
